app/utils/cache.server.ts

```diff
@@ -161,29 +162,12 @@ export async function cachified<Value>({
 	request?: Request
 	timings?: Timings
 }): Promise<Value> {
```

Comment:
Now that the implementation is this easy it might be more straight forward to remove the wrapper altogether in favor of something like

```ts
import { cachified } from 'cachified'
import { baseLruCacheOpts } from '~/utils/cache.server'

cachified({ ...baseLruCacheOpts, key: 'asdf', getFreshValue });
```

But thats hard to decide on before this is actually being used somewhere in the app :)

Also I was unsure why the request is passed here. I remember that on your website the cache was disabled for admins right? Do you have plans to use request in here?

Reply:
Oh, I forgot your comment here. Ummm... Let's remove it.

Reply:
Should we even create the `defaultReporter`?

with the situation that we want to be able to disable the verbose reporter it's probably a good start to go with 

```ts
import { cachified, verboseReporter, mergeReporters } from 'cachified'
import {  lruCache } from '~/utils/cache.server'
import { timingReporter } from '~/utils/timings.server'

cachified({
  cache: lruCache,
  reporter: mergeReporters(timingReporter(timings), verboseReporter()),
  key: 'asdf',
  getFreshValue
});
```

and then maybe create abstractions later when we know more about the common usages.

## Alternative 1: what I proposed above, but with option to disable verbose

```ts
import { cachified } from 'cachified'
import { 
  lruCache,
  defaultReporter // (t) => mergeReporters(timingReporter(t), verboseReporter())
} from '~/utils/cache.server'

const verbose = true;
cachified({ cache: lruCache, reporter: defaultReporter({ timings, verbose }), key: 'asdf', getFreshValue });
```

## Alternative 2: create option sets for lru and db caches

```ts
import { cachified } from 'cachified'
import {  lruCacheOpts, dbCacheOpts } from '~/utils/cache.server'

cachified({
  ...lruCacheOpts({ timings, reporter: null }),
  key: 'asdf',
  getFreshValue
});
```

Reply:
Sorry, I misspoke, I meant let's remove `request` but keep our little wrapper with default options.

Reply:
Ah ok, that makes sense! The timings reporter already is a [no-op when no timings object is passed](https://github.com/epicweb-dev/epic-stack/pull/31/files#diff-9918f0627f11a6c837483de1b259a0deab76d827522275110c45e1f86d069fadR69).

Gonna remove the `request`. And ping you for a final review.